src/DynamoCoreWpf/Controls/InstalledPackagesControl.xaml

```diff
@@ -67,13 +67,24 @@
                                                                MinWidth="50"
                                                                FontSize="11"
                                                                Foreground="{StaticResource PreferencesWindowFontColor}"></TextBlock>
-                                                <TextBlock Text="{x:Static p:Resources.InstalledPackageViewPendingInstallButton}"
-                                                               Visibility="{Binding Path=Model.EnableOldMarkedForUnistallState, Converter={StaticResource BooleanToVisibilityCollapsedConverter}}"
-                                                               FontSize="11"
-                                                               FontStyle="Italic"
-                                                               MinWidth="160"
-                                                               TextAlignment="Center"
-                                                               Foreground="Red"></TextBlock>
+                                                <Grid Height="18"
+                                                    MaxWidth="150">
+                                                    <Border Background="DimGray"
+                                                        HorizontalAlignment="Stretch"
+                                                        CornerRadius="5"
+                                                        ToolTip="{Binding Path=PackageLoadStateTooltip}"
+                                                        Margin="5,0,0,0"
+                                                        Padding="5,0,5,0">
+                                                        <TextBlock Text="{Binding Path=PackageLoadStateText}"
+                                                                TextTrimming="CharacterEllipsis"
+                                                                FontSize="10"
+                                                                FontFamily="Artifakt Element"
```

Comment:
font still does not look quite right

Reply:
is Artifakt actually being used?

Reply:
does not look like it from the UI snapshot.
But no parts of the new UI seem to actually use Artifakt...